src/styles/blocks/navigation.scss

```
@@ -0,0 +1,47 @@
+.nav {
+  margin: 0;
+
+  &__list {
+    display: flex;
+  }
+  &__item {
+    list-style: none;
+    display: flex;
+    padding-left: 20px;
+  }
+  &__item:first-child {
+    padding-left: 0;
+  }
+  &__link {
+    padding: 23px 0;
+    text-decoration: none;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 14px;
+    text-transform: uppercase;
+    color: $color-main-accent;
+    display: flex;
+    align-items: center;
+    cursor: pointer;
+  }
+  &__link.is-active {
+    position: relative;
+  }
+  &__link:hover {
+    color: $color-accent-blue;
+  }
+}
+
+.is-active.nav__link {
```

```suggestion
.is-active {
```
Don't increase specifity